opm/material/fluidsystems/blackoilpvt/ConstantCompressibilityOilPvt.hpp

```diff
@@ -181,8 +181,8 @@ class ConstantCompressibilityOilPvt
                                   const Evaluation& pressure) const
     {
         // Eclipse calculates the viscosity in a weird way: it
-        // calcultes the product of B_w and mu_w and then divides the
-        // result by B_w...
+        // calcultes the product of B_o and mu_o and then divides the
+        // result by B_o...
```

Comment:
I agree. I suppose that this comment is a remant of the time where the viscosity function used the 1/(b*mu) tables directly. I've removed these comments in the latest revision of the patch.